### PR TITLE
Create profile-single-tenant-cluster.md

### PR DESCRIPTION
**Pull request by tashimi**
Signed-off-by: Tasha Drew <tasha.drew@gmail.com>

### CODE REVIEW

File: docs/profiles/profile-single-tenant-cluster.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Kubernetes Multi-tenancy profile: Single Tenant Cluster
+
+In order to define a secure multi-tenant cluster, we must first agree upon a baseline for a secure single tenant cluster. 
+
+## Overall functional model summary
+
+1. A **Tenant** is defined as a team of users/ identities that shall have exclusive use of one or more resources of a Kubernetes cluster in parallel with users of other tenants using their own separate and isolated resources within the same cluster. We will interchangeably use the terms \"users\" or \"members\" of a tenant.
```
Comment (easeway) [inline docs/profiles/profile-single-tenant-cluster.md]: Technically, as called out in [Tenant Concept in Kubernetes](https://docs.google.com/document/d/1ddx7UAEPKFPldBh_diksYO4WZXSHDUhm-e6hyNNGYVU/edit#heading=h.c0uts5ftkk58), defining Tenant as users is ambiguous and problematic. I think I understand overall what this paragraph means and I agree with that. However, is it possible avoid explicitly defining a tenant as a group of users? I recommend saying "A tenant is defined as a group of resources to be exclusively used by a team of users/identities" and avoid the interchangeable use of terms of users/members with tenants. The point is to define tenant from resource perspective instead from users perspective to reduce the ambiguity. My two cents.

File: docs/profiles/profile-single-tenant-cluster.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+2. In this single tenant profile, a Kubernetes cluster and all of its objects and resources are explicitly owned only by a single tenant. There is no sharing of the cluster or attempt to wall off objects and resources between different tenant users.
+
+## Applicable benchmarks used to define a secure Kubernetes cluster
+
+1. NIST SP 800-190: Application Container Security Guide https://csrc.nist.gov/publications/detail/sp/800-190/final 
+1. CIS Benchmark for Kubernetes 1.4 https://csrc.nist.gov/publications/detail/sp/800-190/final
+
+## Kubernetes cluster configuration requirements for profile single tenant
+
+1. The recommended minimum version of Kubernetes release is 1.15. Earlier kubernetes releases may be used with some potential restrictions.  (Note: Maybe better to list api versions required for various Kubernetes resources/ apis).
```
Comment (rjbez17) [inline docs/profiles/profile-single-tenant-cluster.md]: A lot of the features required below are still in beta. May want to call that out specifically that we are recommending non-GA APIs and why it's ok.

File: docs/profiles/profile-single-tenant-cluster.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# Kubernetes Multi-tenancy profile: Single Tenant Cluster
+
+In order to define a secure multi-tenant cluster, we must first agree upon a baseline for a secure single tenant cluster. 
+
+## Overall functional model summary
+
+1. A **Tenant** is defined as a team of users/ identities that shall have exclusive use of one or more resources of a Kubernetes cluster in parallel with users of other tenants using their own separate and isolated resources within the same cluster. We will interchangeably use the terms \"users\" or \"members\" of a tenant.
+
+2. In this single tenant profile, a Kubernetes cluster and all of its objects and resources are explicitly owned only by a single tenant. There is no sharing of the cluster or attempt to wall off objects and resources between different tenant users.
```
[the remark this answers is not in the document]
Comment (tashimi) [reply docs/profiles/profile-single-tenant-cluster.md]: How would you differentiate the two?

File: docs/profiles/profile-single-tenant-cluster.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+1. CIS Benchmark for Kubernetes 1.4 https://csrc.nist.gov/publications/detail/sp/800-190/final
+
+## Kubernetes cluster configuration requirements for profile single tenant
+
+1. The recommended minimum version of Kubernetes release is 1.15. Earlier kubernetes releases may be used with some potential restrictions.  (Note: Maybe better to list api versions required for various Kubernetes resources/ apis).
+1. Kubernetes Role Based Access Control RBAC must be supported and enabled.
+    1. Set up cluster roles 
+    1. Give each user the least permissive role possible to achieve their desired outcome 
+    1. Do not give `write` access to `kubesystem` or other centrally managed namespaces
+    1. Prefer `Roles` and `RoleBindings` to `ClusterRoles` and `ClusterRoleBindings` because they are scoped to a namespace
```
[the remark this answers is not in the document]
Comment (tashimi) [reply docs/profiles/profile-single-tenant-cluster.md]: Yup, I think that's what I'm saying here (don't use cluster roles)

**Pull request merged**